App.tsx

```diff
@@ -13,6 +13,7 @@ import HomeScreen from './screens/HomeScreen';
 import LoginScreen from './screens/LoginScreen';
 import WelcomeScreen from './screens/WelcomeScreen';
 import SignUpScreen from './screens/SignUpScreen';
+import DemoScreen from './screens/DemoScreen';
```

Comment:
`DemoScreen` is imported but not actually added to the login navigation stack. See these lines: https://github.com/ashenafee/STG/blob/13de0aa5d4436a12b05a528d9b1a5dcf92ac5400/App.tsx#L121-L123